patches/server/0141-Basic-PlayerProfile-API.patch

```diff
@@ -111,7 +111,7 @@ index 0000000000000000000000000000000000000000..b7da8606e24b216b39020130fd2c42c7
 +    public UUID setId(@Nullable UUID uuid) {
 +        final GameProfile previousProfile = this.profile;
 +        final UUID previousId = this.getId();
-+        this.profile = new GameProfile(previousProfile.getId(), previousProfile.getName());
++        this.profile = new GameProfile(uuid, previousProfile.getName());
```

Comment:
If the `uuid` argument is null, you should use `Util.NIL_UUID`. The GameProfile constructor does not accept null for either argument.

Reply:
yeah sorry about that